python/trifinger_simulation/visual_objects.py

```diff
@@ -1,3 +1,4 @@
+import numpy as np
 import pybullet
 
 
```

Comment:
I think it would make sense to call this SphereMarker given the convention for naming for the other classes. Also, this class could derive from `ObjectMarker` to avoid redundancy and confusion.